transit_odp/templates/pages/accessibility.html

```diff
@@ -216,25 +217,41 @@ <h2 class="govuk-heading-m">
 
             <section>
                 <h2 class="govuk-heading-m" >{% filter force_escape %}{% trans 'Non-accessible content' %}{% endfilter %}</h2>
-                <h3 class="govuk-heading-s govuk-!-font-size-16">{% filter force_escape %}{% trans 'Content that’s not within the scope of the accessibility regulations' %}{% endfilter %}</h3>
+                <p class="govuk-body govuk-!-font-weight-bold">
+                    {% filter force_escape %}
+                        {% trans 'The content listed below is non-accessible for the following reasons.' %}
+                    {% endfilter %}
+                </p>
+                <p class="govuk-body">
+                    {% filter force_escape %}
+                        {% trans 'Non-compliance with accessibility regulations' %}
+                    {% endfilter %}
+                </p>
                 <p class="govuk-body">
                     <ul class="govuk-list govuk-list--bullet">
                         <li>
                             {% filter force_escape %}
                                 {% blocktrans %}
-                                    Our maps cannot be directly read
-                                    by a screen reader (but key
-                                    information is in the text and
-                                    enlarged print versions can be
-                                    provided)
+                                    Maps cannot be directly read
+                                    by a screen reader. Key
+                                    information is in the text.
```

Comment:
The design says. Please change.
'Maps **functions** cannot be directly read by a..'